library/src/com/astuetz/PagerSlidingTabStrip.java

```diff
@@ -633,6 +633,22 @@ public int getTabPaddingLeftRight() {
         return mTabPadding;
     }
 
+     public LinearLayout getmTabsContainer() {
```

Comment:
No need for the letter 'm' in the getter methods. This is only valid for the variable sintaxis in the class. So instead of `getmTabsContainer` it would be better to call the method `getTabsContainer `